src/test/resources/db/prldb.sql

```diff
@@ -47,8 +47,8 @@ CREATE TABLE public.institutions (
 
 CREATE TABLE public.harvestjobs (
     id SERIAL PRIMARY KEY,
-    institutionID INT,
-    repositoryBaseURL TEXT NOT NULL,
+    institutionId INT,
```

Comment:
Is the upcasing of the last word (something we do in Java) problematic for psql?

Reply:
I think I can answer: I had written the OpenAPI schemas without upcasing the last word, and the ticket specified that those schemas should be matched. I'm fine with either form for JSON fields.

Reply:
(To directly answer your question @ksclarke: no)

Reply:
I personally think consistency between the two would be better if there aren't technical problems with that.

Reply:
Tweaked db script to make field names consistent

Reply:
I think what @ksclarke meant is to have all-uppercase acronyms when they appear at the end of a name, like `institutionID`.

Reply:
Mine definitely are today 😅

Reply:
😁

Reply:
Switched to all caps for ID and URL